lib/parseOptions.js

```diff
@@ -57,7 +57,7 @@ export default function parseOptions (params) {
 
   const proxySimpleExp = /.+:\d+/
   const proxyAuthExp = /.+:.+@.+:\d+/
-  if (options.proxy && !(proxySimpleExp.test(options.proxy) || proxyAuthExp.test(options.proxy))) {
+  if (typeof options.proxy === 'string' && options.proxy && !(proxySimpleExp.test(options.proxy) || proxyAuthExp.test(options.proxy))) {
```

Comment:
Can you duplicate this to https://github.com/contentful/contentful-export/blob/master/lib/parseOptions.js#L44 plz? kthxbye